src/factories/artifact.ts

```diff
@@ -24,8 +22,6 @@ export async function artifactFactory(artifact: Artifact) {
   content.addChild(label)
   element.addChild(content)
 
-  cull.add(element)
```

Comment:
Since artifacts are attached to nodes, culling happens on that level. It's important to not happen here since resizing the x-axis recalculates node widths, and if we cull here then they'll have a width of 0 and won't be placed properly on the newly resized node if it's out of view.